### PR TITLE
Added the tutorial 'Creating your JavaScript application using alfresco-js-api'.

### PR DESCRIPTION
**Pull request by fcorti**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [*] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [*] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
N/A

**What is the new behaviour?**
N/A

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [*] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
Added the tutorial 'Creating your JavaScript application using alfresco-js-api'.

### CODE REVIEW

File: docs/tutorials/creating-javascript-app-using-alfresco-js-api.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+---
+# Creating your Alfresco JavaScript application
+
+In this tutorial you are going to see how to create from scratch an application in JavaScript to interact with Alfresco. This is a "getting started" task that should enable you to start developing your own JavaScript application on top of Alfresco Content Services or Alfresco Process Services.
+
+In this tutorial you are going to learn how to develop on top of Alfresco Content Services, but the development on top of Alfresco Process Services follows exactly the same principle.
+
+**Note:** The development on top of Alfresco Content Services AND Alfresco Process Services has the only limitation of the [CORS](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing). In this case you might need to proxy your application first.
+
+## Prerequites
```
Comment (therealandeeee) [inline docs/tutorials/creating-javascript-app-using-alfresco-js-api.md]: Should be "prerequisites"

File: docs/tutorials/creating-javascript-app-using-alfresco-js-api.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+If you don't have an instance of Alfresco Content Services up and running check the [preparation of the development environment](./preparing-the-development-environment.html).
+
+Only to download the requested libraries, you will need the `npm` client. Also in this case check the [preparation of the development environment](./preparing-the-development-environment.html) for further details on how to install it into your development environment.
+
+## Creating the JavaScript application
+
+Assuming that you have your Alfresco Content Services instance up an running at `http://localhost:8082/alfresco`, let' see here how to develop a JavaScript application from scratch. The JavaScript application will be able to interact with Alfresco back-end services using the [`alfresco-js-api`](https://github.com/Alfresco/alfresco-js-api) library. This library does not necessarily require to be used into an Angular application but it is "framework agnostic".
```
Comment (therealandeeee) [inline docs/tutorials/creating-javascript-app-using-alfresco-js-api.md]: "up *and* running..." and "let'*s* see here.."

File: docs/tutorials/creating-javascript-app-using-alfresco-js-api.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+This is all you need to deploy the JavaScript application into the Tomcat instance of the container of Alfresco Content Services.
+
+## The JavaScript application in action
+
+To see the JavaScript application in action, open a browser at `http://localhost:8082/my-js-app` and you should see something like the following screenshot.
+
+![javascript_app_launch](../docassets/images/javascript_app_launch.png)
+
+Of course this is a very basic example, used to show in practice how to develop from scratch a JavaScript application (not necessarily and Angular application) interacting with Alfresco Back-end Services using the [`alfresco-js-api`](https://github.com/Alfresco/alfresco-js-api) library.
```
Comment (therealandeeee) [inline docs/tutorials/creating-javascript-app-using-alfresco-js-api.md]: "...*an* Angular application"

File: docs/tutorials/creating-javascript-app-using-alfresco-js-api.md (hunk 1 above)
Comment (fcorti) [reply docs/tutorials/creating-javascript-app-using-alfresco-js-api.md]: Changed!

File: docs/tutorials/creating-javascript-app-using-alfresco-js-api.md (hunk 2 above)
Comment (fcorti) [reply docs/tutorials/creating-javascript-app-using-alfresco-js-api.md]: Changed!

File: docs/tutorials/creating-javascript-app-using-alfresco-js-api.md (hunk 3 above)
Comment (fcorti) [reply docs/tutorials/creating-javascript-app-using-alfresco-js-api.md]: Updated!

**Comment by fcorti** [comment]
Thanks for the fast (and valuable) review @therealandeeee.

**Pull request merged**